library/templates/nysdec_solidwaste.yml

```diff
@@ -0,0 +1,32 @@
+dataset:
+  name: &name nysdec_solidwaste
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://data.ny.gov/api/views/2fni-raj8/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
```

Comment:
is geometry type is none, set `SRS: null`, we couldn't do this before but now we can